pages/solid-events.md

```diff
@@ -23,7 +23,7 @@ Are you interested in speaking about what you have been building on Solid? Reach
 |----|-------|-------------|------------------------------|
 | 2021-05-27 | [Your Personal Linked Data Graphs with Solid](http://www.lotico.com/index.php/Your_Personal_Linked_Data_Graphs_with_Solid)|[Lotico](http://www.lotico.com)||
 | 2021-05-07 | [Understory Digital Garden Club — Solid](https://www.eventbrite.com/e/understory-digital-garden-club-tickets-151311208899)|[Understory](https://www.understory.coop/)||
-| 2021-05-06 | [Solid World May](https://www.eventbrite.com/e/solid-world-may-2021-tickets-146761614941)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/546202189)<br/>**Jackson Morgan’s Links** Liqid Chat Demo: https://tinyurl.com/liqid-chat, Twitter: https://twitter.com/otherJackson, YouTube: https://www.youtube.com/channel/UCMVpUX6URfy5SnRn9OWteyQ, LinkedIn: https://www.linkedin.com/in/jacksoncmorgan/, TikTok: https://www.tiktok.com/@otherotherjackson, Client Repo: https://github.com/o-development/ochat-client API Repo: https://github.com/o-development/ochat-api **Christian Buggedei’s Links** https://darcy.is/, https://opencollective.com/darcy-is/updates/we-are-looking-for-help, **Aaron Coburn's and Jack Lawson's Links** https://notifications-demo.docs.inrupt.com, https://podbrowser.inrupt.com, https://docs.inrupt.com/ess/latest/services/service-websocket/|
+| 2021-05-06 | [Solid World May](https://www.eventbrite.com/e/solid-world-may-2021-tickets-146761614941)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/546202189)<br/>Jackson Morgan’s Links [Liqid Chat Demo](https://tinyurl.com/liqid-chat), [Twitter](https://twitter.com/otherJackson), [YouTube](https://www.youtube.com/channel/UCMVpUX6URfy5SnRn9OWteyQ), [LinkedIn](https://www.linkedin.com/in/jacksoncmorgan/), [TikTok](https://www.tiktok.com/@otherotherjackson), [Client Repo](https://github.com/o-development/ochat-client) [API Repo](https://github.com/o-development/ochat-api) Christian Buggedei's Links [Darcy.is](https://darcy.is/), [Help Needed](https://opencollective.com/darcy-is/updates/we-are-looking-for-help), Aaron Coburn's and Jack Lawson's Links[Notifications Demo](https://notifications-demo.docs.inrupt.com), [PodBrowser](https://podbrowser.inrupt.com), [Docs](https://docs.inrupt.com/ess/latest/services/service-websocket/)|
```

Comment:
There's a space missing after Aaron and Jack's links. Also suggested some additional punctuation :)

```suggestion
| 2021-05-06 | [Solid World May](https://www.eventbrite.com/e/solid-world-may-2021-tickets-146761614941)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/546202189)<br/>Jackson Morgan’s links: [Liqid Chat Demo](https://tinyurl.com/liqid-chat), [Twitter](https://twitter.com/otherJackson), [YouTube](https://www.youtube.com/channel/UCMVpUX6URfy5SnRn9OWteyQ), [LinkedIn](https://www.linkedin.com/in/jacksoncmorgan/), [TikTok](https://www.tiktok.com/@otherotherjackson), [Client Repo](https://github.com/o-development/ochat-client) [API Repo](https://github.com/o-development/ochat-api). Christian Buggedei's links: [Darcy.is](https://darcy.is/), [Help Needed](https://opencollective.com/darcy-is/updates/we-are-looking-for-help). Aaron Coburn's and Jack Lawson's links: [Notifications Demo](https://notifications-demo.docs.inrupt.com), [PodBrowser](https://podbrowser.inrupt.com), [Docs](https://docs.inrupt.com/ess/latest/services/service-websocket/).|
```